packages/KSelect/KSelect.vue

```diff
@@ -295,6 +295,7 @@ export default {
       })
       this.filterStr = this.appearance === 'dropdown' ? '' : item.label
       this.$emit('selected', item)
+      this.$emit('input', item)
```

Comment:
Since you're adding an emit for the `input` event, should you add an emit on `change` as well since `change` and `input` are both events fired by `<select>` elements?

Reply:
Good idea, I'll add it!

Reply:
It looks like you also need to update the Events section on `select.md` for all new events

Reply:
Fixed!